curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-122-efficient-exponentiation.md

```diff
@@ -39,23 +39,23 @@ For $1 ≤ k ≤ 200$, find $\sum{m(k)}$.
 
 # --hints--
 
-`efficientExponentation()` should return `1582`.
+`efficientExponentiation()` should return `1582`.
```

Comment:
The changes in this file should be reveresed

Reply:
I assume we did - AFAICT, Euler doesn't even care about functions.